my-react-app/src/features/user/search/searchbar.js

```diff
@@ -0,0 +1,143 @@
+//import './searchbar.css'
+import {Container, InputGroup, FormControl, Button, Row, Card, Dropdown} from 'react-bootstrap';
+import {useState, useEffect} from 'react';
+
+const CLIENT_ID = "2f6e085b55bc4ede9131e2d7d7739c30";
+const CLIENT_SECRET = "88eeb98034e5422099cce4f6467a3d51";
+
+function Search() {
```

Comment:
the current industry standard syntax for ES6 functional components would be: const Search = () => {
// Rest of Code here
}

Functionally there is no difference but its good to follow 1 coding standard